frontend/app/actions/ActionRefiners.scala

```diff
@@ -70,7 +70,7 @@ object ActionRefiners extends LazyLogging {
       authRequest = new AuthenticatedRequest(user, request)
       tp = authRequest.touchpointBackend
       member <- OptionEither(authRequest.forMemberOpt)
-      subscription <- OptionEither.liftEither(tp.subscriptionService.either[SubscriptionPlan.FreeMember, SubscriptionPlan.PaidMember](member))
+      subscription <- OptionEither.liftEither(tp.subscriptionService.either[SubscriptionPlan.FreeMember, SubscriptionPlan.PaidMember](member).map(_.toOption.flatten))
```

Comment:
oh, we're already doing a liftEither, so no need to flatten the either out.  Just needed to remove the '.liftEither'

Reply:
that assumes we want to keep the error if it failed, rather than replacing with None.  So the user will get a 500 error on line 88 instead of just logged out.

Reply:
as discussed we will keep the old behaviour rather than surfacing the error that would have been treated as not found